samples/event_manager_proxy/boards/nrf54l15dk_nrf54l15_cpuapp.overlay

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	chosen {
+		/delete-property/ zephyr,bt-hci;
+	};
+
+	soc {
+		reserved-memory {
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			sram_rx: memory@20018000 {
+				reg = <0x20018000 0x0800>;
+			};
+
+			sram_tx: memory@20020000 {
+				reg = <0x20020000 0x0800>;
```

Comment:
As @grochu mentioned: https://github.com/nrfconnect/sdk-nrf/pull/22815/files#r2154289705 
shouldn't this address be 0x20018800?
sram_rx has a size of 0x0800, not 0x8000